### ISSUE TITLE
Migration to node-red-contrib-amazon-echo

### ISSUE THREAD

**Issue by nnhhuu** [originallyus/node-red-contrib-alexa-local#101]
I'm the author of this node.

**TL;DR;**
Please replace my node with this node [node-red-contrib-amazon-echo](https://flows.nodered.org/node/node-red-contrib-amazon-echo)

------------------------------------------------------------
A few things to confirm:
• My nodes is **no longer working** with **any** Alexa devices after recent updates.
• The reason is discovery procedure needs to be hosted on port 80 of your NodeRed.
• Newer generation of Alexa devices are not working with my node for the same reason.
• I recommend switching over to this node [node-red-contrib-amazon-echo](https://flows.nodered.org/node/node-red-contrib-amazon-echo). It is basically a clone of my node, with a separate Node for discovery, running on port 80 by default.

**VERY IMPORTANT NOTE:**
• You probably need to restart the system after replacing my nodes to clear them from the system.
• You need to make sure **your port 80 is free** & allowed to be used by NodeRED. You can either:
• Run NodeRED as root user, which allow you to open any port on the system.
OR
• Use iptables to redirect xxxxx port to 80. Instructions are available [here](https://github.com/datech/node-red-contrib-amazon-echo/wiki/Troubleshooting).

This hassle is exactly why I have designed my node to run on random ports, pretty newbie-proof for most users & did not change the implementation for newer generation of Alexa devices.

_Originally posted by @torinnguyen in https://github.com/originallyus/node-red-contrib-alexa-local/issues/100#issuecomment-531777775_

**Comment by nnhhuu** [comment]
first of all, thank you for your work, i've used node-red-contrib-alexa-local intensively for so many months.

I'm having a problem with the migration to node-red-contrib-amazon-echo, after I uninstalled the node-red-contrib-alexa-local  and forgot all the devices,  my echo's still discovering the old devices, i mean when i discover  devices, it shows always two devices, one form the old node (node-red-contrib-alexa-local) and one for the new node (node-red-contrib-amazon-echo).   How can i disappear the reference to the old devices.

Thank you in advance for your help.

**Comment by nThanksForAllTheFish** [comment]
Thank you for node-red-contrib-alexa-local which has worked for me for many months.  After switching to  node-red-contrib-amazon-echo - as root - the node works but Alexa responds "I'm not sure what went wrong".  Any idea why and how to fix this?

**Comment by tdejneka** [comment]
>  my echo's still discovering the old devices, i mean when i discover devices, it shows always two devices

Did you ever find a solution?
